src/databricks/labs/ucx/mixins/sql.py

```diff
@@ -78,7 +78,7 @@ def __init__(self, ws: WorkspaceClient):
         self.type_converters = {
             ColumnInfoTypeName.ARRAY: json.loads,
             # ColumnInfoTypeName.BINARY: not_supported(ColumnInfoTypeName.BINARY),
-            ColumnInfoTypeName.BOOLEAN: bool,
+            ColumnInfoTypeName.BOOLEAN: lambda value: eval(value.capitalize()),
```

Comment:
I need this to be function, to explicitly test if a string value is 'true' or 'false'

Reply:
Don't eval() anything, it's a security hole. No way we're allowing security holes.

Lowercase & compare to "true".

Reply:
Got it, didn't know about the security risk from eval, thanks for the docs